lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -332,9 +332,10 @@ swift::checkGlobalActorAttributes(SourceLoc loc, DeclContext *dc,
 
     // Only a single global actor can be applied to a given entity.
     if (globalActorAttr) {
-      ctx.Diags.diagnose(
-          loc, diag::multiple_global_actors, globalActorNominal->getName(),
-          nominal->getName());
+      ctx.Diags
+          .diagnose(loc, diag::multiple_global_actors,
+                    globalActorNominal->getName(), nominal->getName())
+          .fixItRemove(globalActorAttr->getRangeWithAt());
```

Comment:
In cases were it's ambiguous which attribute was intended the compiler intentionally doesn't provide a fix-it, user has to decide which one they want to keep/remove.